packages/sitecore-jss-vue/src/components/RichText.test.ts

```diff
@@ -111,4 +112,85 @@ describe('<RichText />', () => {
     expect(rendered.attributes()).toMatchObject(attrs);
     expect(rendered.element.innerHTML).toBe(props.field.value);
   });
+
+  it('should initialize links', () => {
+    const props = {
+      field: {
+        value: '<div id="test"><h1>Hello!</h1><a href="/t10">1</a><a href="/t10">2</a></div>',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    expect(rendered.element.innerHTML).toContain('<div id="test">');
+    expect(rendered.element.innerHTML).toContain('<h1>Hello!</h1>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">1</a>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">2</a>');
+  });
+
+  it('should not initialize links when does not have value', () => {
+    const props = {
+      field: {
+        value: '',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    const hasChildNodes = rendered.element.hasChildNodes();
+    expect(hasChildNodes).toBe(false);
+  });
+  it('should not initialize links if no links in markup', () => {
+    const props = {
+      field: {
+        value: '<div id="test"><h1>Hello!</h1></div>',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    expect(rendered.element.innerHTML).toContain('<div id="test">');
+    expect(rendered.element.innerHTML).toContain('<h1>Hello!</h1>');
+    expect(rendered.element.innerHTML).not.toContain('<a href=');
+  });
+  it('should navigate if anchor element clicked', async () => {
```

Comment:
This test can be renamed to -> `Should initialize links`

Reply:
And inside this test you can check both cases, when you have nested element and link, so you can click on nested, after that on parent and check it here